src/mousev1_to_nwb/abdeladim_2023/abdeladim_2023imagingextractor.py

```diff
@@ -0,0 +1,162 @@
+from typing import Optional, Tuple, List, Iterable
+from pathlib import Path
+import glob
+from roiextractors.extractors.tiffimagingextractors.scanimagetiffimagingextractor import (
+    ScanImageTiffMultiPlaneImagingExtractor,
+    ScanImageTiffSinglePlaneImagingExtractor,
+)
+
+from roiextractors.imagingextractor import ImagingExtractor
+from roiextractors.multiimagingextractor import MultiImagingExtractor
+from neuroconv.utils import FolderPathType
+from roiextractors.extraction_tools import PathType, FloatType, ArrayType, DtypeType, get_package
+import numpy as np
+
+
+class Abdeladim2023MultiPlaneImagingExtractor(ImagingExtractor):
+    """Specialized extractor for Abdeladim2023 conversion project: reading ScanImage .tif files chunked over time"""
+
+    extractor_name = "Abdeladim2023MultiPlaneImagingExtractor"
+    is_writable = True
+    mode = "folder"
+
+    def __init__(
+        self,
+        folder_path: FolderPathType,
+        channel_name: Optional[str] = None,
+    ) -> None:
+        self.folder_path = Path(folder_path)
+        tif_file_paths = sorted(glob.glob(f"{self.folder_path}/*.tif"))
```

Comment:
You can do Path(folder_path).glob(...) no need to import glob

Reply:
I will change this in #4

Reply:
I will change this in #54